tests/test_pipeline_output.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+import numpy as np
+import os
+import allensdk.core.json_utilities as ju
+
+
+import sys
+sys.path.append("/local1/pyprojects/allensdk.ipfx/bin") # there is got to be a better way
```

Comment:
move the `bin` directory to `allensdk.ipfx/allensdk/ipfx/bin` and add an `__init__.py`.  Then you can do this:

```python
from allensdk.ipfx.bin.run_pipeline import run_pipeline
```